plugin-flex-ts-template-v2/src/feature-library/supervisor-capacity/config.ts

```diff
@@ -0,0 +1,12 @@
+import { getFeatureFlags } from '../../utils/configuration';
```

Comment:
General question about the `config.ts` convention -- why do some features have a `config.ts` at their root folder but others don't? It looks like the code to check if the feature is enabled (`if (!isFeatureEnabled()) return {};`) is sometimes in `index.ts` and other times in the `config.ts`.

Reply:
This was to prevent cyclical imports when a flex hook uses config (index.ts imports all flex-hooks, so a flex-hook importing index to get config makes a big sad). I'll make this more consistent; we should establish the pattern up front so others dont run into issues.